src/exporters/crypto_init_tests.rs

```diff
@@ -0,0 +1,13 @@
+use std::sync::Once;
+
+static INIT_CRYPTO: Once = Once::new();
+
+// Allowing dead_code because this is for unit tests ONLY!
+#[allow(dead_code)]
```

Comment:
I'd be curious if you could use the `#[cfg(test)]` macro here to indicate it's for tests

Reply:
You can, it's more verbose but went ahead and did it

Reply:
Seems worth it though as you can remove the comment because it's explicit